pom.xml

```diff
@@ -555,12 +568,6 @@
             <artifactId>jackson-datatype-jsr310</artifactId>
             <version>${jackson.version}</version>
         </dependency>
-        <!-- Jackson modules to serialize Jersey response objects to JSON. -->
-        <dependency>
-            <groupId>com.fasterxml.jackson.jaxrs</groupId>
-            <artifactId>jackson-jaxrs-json-provider</artifactId>
```

Comment:
Check whether this is still being updated

Reply:
https://github.com/FasterXML/jackson-jaxrs-providers/blob/2.13/json/src/main/java/com/fasterxml/jackson/jaxrs/json/JacksonJsonProvider.java and https://github.com/eclipse-ee4j/jersey/blob/master/media/json-jackson/src/main/java/org/glassfish/jersey/jackson/internal/jackson/jaxrs/json/JacksonJsonProvider.java seem to be copies of each other

Reply:
I know the jackson library used to be much faster then the reference implementation; Hence most project would use the jackson lib for that reason, but I am not up to date on the subject so we should check before we switch library.

Reply:
If they are equivalent, copies then we can proceed.